packages/dashboard-frontend/src/containers/FactoryLoader/updateDevfileMetadata.ts

```diff
@@ -49,6 +51,18 @@ export default function updateDevfileMetadata(
     if (createPolicy !== 'peruser' && metadata.name) {
       metadata.name = `${metadata.name}-${getRandomString(4).toLowerCase()}`;
     }
+
+    if (storageType === 'ephemeral') {
+      if (devfile.schemaVersion === '2.0.0') {
```

Comment:
wondering if we should also handle 2.1.0 and 2.2.0 ?

Reply:
No, because from version 2.1.0 and above, "attributes" have been moved from metadata to the root and that is it.